COLLABORATOR_GUIDE.md

```diff
@@ -56,16 +56,23 @@ full responsibility for the change, defer to another Collaborator.
 Before landing pull requests, sufficient time should be left for input
 from other Collaborators. Leave at least 48 hours during the week and
 72 hours over weekends to account for international time differences
-and work schedules. Trivial changes (e.g. those which fix minor bugs
-or improve performance without affecting API or causing other
-wide-reaching impact) may be landed after a shorter delay.
+and work schedules. Trivial changes (e.g. fixing typographical errors in
+documentation) may be landed after a shorter delay.
```

Comment:
What is the reason for removing «fixing small bugs» here? Does that mean that those also should receive 48 hours for collecting the reviews?